tap_zoom/schemas/webinar_poll_results.json

```diff
@@ -40,6 +40,19 @@
               "null",
               "string"
             ]
+          },
+          "date_time": {
+            "type": [
+              "null",
+              "string"
+            ],
+            "format": "date-time"
+          },
+          "polling_id": {
+            "type": [
+              "null",
+              "string"
+            ]
```

Comment:
are the `start_time` and `webinar_id` fields not required for this stream

Reply:
https://developers.zoom.us/docs/api/rest/reference/zoom-api/methods/#operation/listPastWebinarPollResults

Reply:
No, as you can see in endpoints.py file, for webinar_poll_results path data_key is `questions` and those are only the part of api response we are storing.